lib/containers/utils.pm

```diff
@@ -179,4 +179,22 @@ sub test_built_img {
     assert_script_run("rm -rf /root/templates");
 }
 
+=head2 can_build_sle_base
+
+C<can_build_sle_base> should be used to identify if sle base image runs against a
+system that it does not support registration and SUSEConnect B<for job running on OSD>.
+In this case the build of the container engine is not going to work.
+
+The call should return false if you try to test sle base container on osd but host is not sle
+true otherwise.
+
+=cut
+sub can_build_sle_base {
+    my $is_osd = get_var('OPENQA_HOSTNAME', '') =~ /suse.de|suse.cz/;
```

Comment:
Using `OPENQA_HOSTNAME` to change the test flow is just fundamentally broken. It makes local runs impossible.

Reply:
My local instance is suse.cz. i havent included the `.asia` because i was not aware of it. i actually pumped on this problem my self before i add suse.cz. But i think it is not big problem because we always do *hacks* local when we run a test.

Reply:
same here https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/11397

Reply:
@Vogtinator i did some changes here.